libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -19,12 +26,45 @@ export class StripeManager {
    * @param customerId
    * @returns
    */
-  async fetchActiveCustomer(customerId: string) {
-    const customer = await this.client.fetchCustomer(customerId);
+  async fetchActiveCustomer(
+    customerId: string,
+    params?: Stripe.CustomerRetrieveParams
+  ) {
+    const customer = await this.client.fetchCustomer(customerId, params);
     if (customer.deleted) throw new CustomerDeletedError();
     return customer;
   }
 
+  /**
+   * Retrieves active subscriptions
+   *
+   * @param customerId
+   */
+  async fetchActiveSubscriptions(subscriptions: Stripe.Subscription[]) {
+    return subscriptions.filter(
+      (sub) =>
+        ACTIVE_SUBSCRIPTION_STATUSES.includes(sub.status) &&
+        sub.collection_method === 'send_invoice'
+    );
+  }
```

Comment:
I think this belongs in the paypal manager - `send_invoice` is the collection method used for paypal subs.